webui/react/src/components/InteractiveTable.tsx

```diff
@@ -319,7 +319,8 @@ const InteractiveTable: InteractiveTable = ({
         ?.getPropertyValue('--theme-sizes-layout-big').slice(0, -2),
     );
     if (typeof pagePadding !== 'number') pagePadding = 16;
-    return columnsWidths.reduce((a, b) => a + b, 0) + 2 * WIDGET_COLUMN_WIDTH + pagePadding;
+    const padding = 24;
+    return columnsWidths.reduce((a, b) => a + b, 0) + 2 * WIDGET_COLUMN_WIDTH + 2 * padding ;
```

Comment:
is there any reason to put `padding = 24` here?
do we no longer use pagePadding?

Reply:
this is basically just replicated [this](https://github.com/determined-ai/determined/pull/4138) it's to make the horizontal scrollbars not have to appear.

we can take it out as part of merge.